src/flags/date.rs

```diff
@@ -83,6 +83,25 @@ impl Configurable<Self> for DateFlag {
             None
         }
     }
+
+    /// Get a potential `DateFlag` variant from the environment.
+    fn from_environment() -> Option<Self> {
+        if let Ok(value) = std::env::var("TIME_STYLE") {
+            match value.as_str() {
+                // TODO: time 0.2 supports the %N specifier for nanoseconds, proper format for
```

Comment:
We could bump the version of `time` to latest.

Reply:
Sadly `time 0.2` is a near complete rewrite of `time 0.1`‌ and is not compatible. I've switched to using `chrono` because `chrono-humanize` is being using anyway.